package.json

```diff
@@ -18,7 +18,6 @@
     "font-awesome": "FortAwesome/Font-Awesome#~4.2.0",
     "fontawesome": "FortAwesome/Font-Awesome#~4.2.0",
     "glfx-js": "jywarren/glfx.js#*",
-    "image-sequencer": "publiclab/image-sequencer#~1.4.0",
```

Comment:
Attempt to install this dependency gives the following output:

![image](https://user-images.githubusercontent.com/19597045/60685619-689f9b80-9e7a-11e9-9962-7d651c6e72e7.png)

Reply:
Huh this is recent. Wasn't this working a while ago?

Reply:
When you switched to Yarn in https://github.com/publiclab/mapknitter/pull/702 we were able to `yarn install` either locally or using docker right? I tried to reset both yarn.lock and pckage.json files to #702 state and I wasn't able to install image-sequencer but was able to install leaflet-environmental-layers. I'm kinda confused with this, tried cleaning the cache and manually installing those but still no success.

Reply:
Yeah this is very odd! I noted in #791 that image-sequencer is fetching from GitHub while LEL is fetching from NPM, not sure if that's an issue. I am able to consistently fetch it manually with `yarn install` after bumping the version number. I don't see the timeouts from my local or manually in the stable/unstable servers when logged in there.

Reply:
I'm not able to fetch it both deps, neither using yarn or npm.